single_cell_RNAseq_snake/profiles/generic/config.v8+.yaml

```diff
@@ -0,0 +1,16 @@
+executor: slurm
```

Comment:
I know the upgrade to v8 changes things, but not sure this should be in the config file names, you mention the README?

Reply:
generally agreed, but from the snakemake docs:

```The profile folder is expected to contain a configuration file that file that defines default values for the Snakemake command line arguments. The file has to be named config.vX+.yaml with X denoting the minimum supported Snakemake major version (e.g. config.v8+.yaml)```